internal/app/connectconformance/connectconformance_test.go

```diff
@@ -100,8 +100,8 @@ func TestRun(t *testing.T) {
 	allPermutations := testCaseLib.allPermutations(true, true)
 	expectedNumCases := len(allPermutations)
 
-	// 208 test cases as of this writing, but we will likely add more
-	require.GreaterOrEqual(t, expectedNumCases, 208)
+	// 194 test cases as of this writing, but we will likely add more
+	require.GreaterOrEqual(t, expectedNumCases, 194)
```

Comment:
I removed some redundant test cases from `basic.yaml`, which resulted in this number falling.